<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duco Ventilation Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="topBoxCombobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DucoTop 60 ZR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TronicTop 60</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="kleurCombobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>F1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PA Duco RAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PA andere RAL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAL (DucoRal)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Andere RAL</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="isolatieCombobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Standaard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AK+</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="lengtSpinbox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="berekenButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="raamCombobox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Corto/Basso</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medio/Alto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Largo/Grando</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="finaalWaardes">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>921</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pbLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>821</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="kamerComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Woonkamer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slaapkamer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Werkruimte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hobbyruimte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Anders</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="klemBord">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>240</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Kopieer naar klembord</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lengte:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aantal:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="aantalSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
